src/pages/ProfilePage/ProfileComponent/ListComponent.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import ArrowForwardIosIcon from "@material-ui/icons/ArrowForwardIos";
+import {
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Typography,
+  Switch as MuiSwitch,
+  makeStyles,
+} from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    "& .MuiTypography-h1": {
+      marginLeft: `${theme.spacing(2)}px`,
+      marginBottom: `${theme.spacing(2)}px`,
+      fontSize: "25px",
+      fontWeight: 600,
+      color: "#0D0F44",
+    },
+    "& .MuiSvgIcon-root": {
+      fontSize: "0.8rem",
+      color: "#120A85",
+    },
+    "& .MuiTypography-body1": {
+      fontSize: "1rem",
+      marginBottom: "5px",
+      color: "#0D0F44",
+      fontWeight: 600,
+    },
+    "& .MuiTypography-body2": {
+      fontSize: "0.8rem",
+      color: "#42607D",
+      fontWeight: 500,
+    },
```

Comment:
I see you have worked hard with providing the exact styles to different typography variant.

Here is the typography configuration: https://github.com/codeforcauseorg/edu-client/blob/main/src/theme/typography.js

If you think that these sizes and weights are not appropriate and is not aligning to use cases then you can create  a new PR that changes these styles.

We don't want developers to do the repetitive things again and that's why we have typography so feel free to change it the most common cases.

I guess the font-sizes mentioned are too small, you can change those it is one of the required things too.

Reply:
Sure @Abhishek-kumar09 I'll create PR for that :)

Reply:
> I guess the font-sizes mentioned are too small, you can change those it is one of the required things too.


is font-size mentioned in the profile page user interface or in the [typography](https://github.com/codeforcauseorg/edu-client/blob/main/src/theme/typography.js) configuration is small?

Reply:
I was talking about typogaphy, if it looks way to small than the actual used ones.

Reply:
Sure @Abhishek-kumar09  got it :)

Reply:
Hello, @Abhishek-kumar09  as @faisaljamil25  is working on the typography I things I can go for removing some extra styling :)